internal/controller/monovertexrollout/monovertexrollout_progressive.go

```diff
@@ -343,13 +331,13 @@ func scaleDownPromotedMonoVertex(
 		"actualPodsCount", actualPodsCount,
 		"newMin", newMin,
 		"newMax", newMax,
-		"originalScaleDefAsString", originalScaleDefAsString,
+		"originalScaleMinMax", originalScaleMinMax,
 	).Debugf("found %d pod(s) for the monovertex, scaling down to %d", actualPodsCount, newMax)
 
 	scaleValuesMap[promotedChildDef.GetName()] = apiv1.ScaleValues{
-		OriginalScaleDefinition: originalScaleDefAsString,
-		ScaleTo:                 newMax,
-		Actual:                  actualPodsCount,
+		OriginalScaleMinMax: &originalScaleMinMax,
```

Comment:
nit: do you still want this to be a pointer to a string or do you want to change it to a simple string type?